app/assets/stylesheets/application/homepage/_event_thumbnail.scss

```diff
@@ -0,0 +1,55 @@
+.event-thumbnail {
+  background-color: $secondary-background-color;
+  vertical-align: top;
+  text-align: left;
+  position: relative;
+  display: inline-block;
+  margin: 20px 7px;
+  width: 292px;
+
+  &:hover {
+    background-color: #ece9e7;
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  img {
+    object-fit: cover;
+    height: 194px;
+    width: 292px;
+  }
+
+  h2 {
+    word-spacing: 2px;
```

Comment:
Properties should be ordered font-family, font-size, line-height, padding, word-spacing